### ISSUE TITLE
Think / Discuss with team about using gitpod more or even accross the board

### ISSUE THREAD

**Issue by kevinhowbrook** [Tolulope/wagtail-workshop#30]

**Comment by kevinhowbrook** [comment]
I've chatted to Tom D about this at torchbox and he thinkis it's a good idea. I'm going to run through it on gitpod this weeked to see how it goes. Tom also offered to help us set up a 'launch on gitpod' button for the docs.

**Comment by Tolulope** [comment]
I agree with using Gitpod! I think it's useful for a quick overview, but there is value in building the site from scratch. Depending on timescales, we can make that (building on the machine) an extra option.

**Comment by kevinhowbrook** [comment]
Yea that's what I was thinking - There can be a section in the docs covering local development. By that time students will be more familiar with things

**Comment by kevinhowbrook** [comment]
Yea that's what I was thinking - There can be a section in the docs covering local development. By that time students will be more familiar with things.
